synfig-studio/src/gui/dialogs/dialog_guide.h

```diff
@@ -0,0 +1,99 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file dialogs/Dialog_Guide.h
+**	\brief Template Header
```

Comment:
Please add a simple description on \brief line. "Template Header" is the example of the... template file XD

Reply:
oops o_O, done